pkgs/development/python-modules/generic/run_setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
```

Comment:
I agree - it's FAR from ideal, but it's the best method out there. I'll add docs about this approach.